tests/python_client/testcases/test_search.py

```diff
@@ -2887,7 +2887,7 @@ def test_search_with_expression_auto_id(self, dim, expression, _async, enable_dy
             assert set(ids).issubset(filter_ids_set)
 
     @pytest.mark.tags(CaseLabel.L2)
-    def test_search_expression_all_data_type(self, nb, nq, dim, auto_id, _async, enable_dynamic_field):
```

Comment:
Suggest not remove the dynamic switch, could modify the function "gen_default_rows_data_all_data_type"

Reply:
> Suggest not remove the dynamic switch, could modify the function "gen_default_rows_data_all_data_type"

I have modified the function.